content/altinn-studio/getting-started/app-dev-course/modul7/_index.nb.md

```diff
@@ -0,0 +1,138 @@
+En bruker som ikke oppfyller kravene for skjemaet skal stoppes så tidlig som mulig i arbeidsflyten. 
+Hvis brukeren ikke bekrefter at skjemaet er relevant, skal de sendes til en ny forklaringsside. Hvis de bekrefter relevansen,
+skal de sendes til datainnsamlingssidene.
+
+### Krav 2: Vis stillingsutlysninger for IKT-bransjen
+Om en bruker velger **IKT (data/it)** under bransje skal det vises en tekst med en lenke til en av deres stillingsutlysninger.
+
+## Implementer dynamisk visning og skjuling av sider
+For å oppfylle krav 1 fra Sogndal kommune, må vi opprette en ny side som skal vises hvis appen ikke er relevant for brukeren. 
+Vi legger også til en funksjon der brukeren kan bekrefte relevansen, og basert på dette valget vil vi dynamisk skjule eller vise sider ved hjelp av logiske uttrykk.
+
+### Legg til ny side
+Vi legger til en ny side som skal vises hvis brukeren ikke bekrefter at de oppfyller kravene for appen.
+1. Klikk på **Utforming**.
+2. Klikk på **Legg til ny side** og gi den navnet ```ikke-for-deg``` 
+3. Dra inn en komponent av typen **Avsnitt**.
+4. Klikk på **Ledetekst** under **Tekst**-menyen til komponenten.
+5. Kopier inn følgende tekst: 
+    ```md
+    Dette skjemaet er ikke for deg.
+    Se en oversikt over andre tilbud i kommunen [her](https://www.sogndal.kommune.no/).
+    ```
+Appen skal nå bestå av fire sider.
+
+!["Fire sider"](fire_sider.png)
+
+### Oppdater datamodellen
+For at appen skal kunne vurdere om skjemaet er relevant for brukeren, må vi oppdatere datamodellen.
+1. Klikk på **Datamodell**.
+2. Klikk på **Legg til**.
+3. Klikk på **Ja/nei**.
+4. Endre navnet på feltet til ```KanBrukeSkjema```.
+5. Klikk **Generer modeller**.
+
+Datamodellen skal nå se slik ut:
+
+!["Datamodell med KanBrukeSkjema"](datamodell.png)
+
+### Legg til avkrysningsbokser
+Vi legger til avkrysningsbokser på infosiden så brukeren kan bekrefte at skjemaet er relevant for dem.
+1. Klikk på **Utforming**.
+2. Klikk på **infoside** og dra inn en komponent av typen **Radioknapper**.
+    !["Skjermbilde av radioknapper-komponenten satt inn i skjemabyggeren"](skjemabygger_med_radioknapper.png)
+3. Klikk på **Ledetekst** under **Tekst**-menyen til komponenten og kopier inn teksten:
+    ```Jeg møter kriteriene for å bruke appen.```
+4. Lag en ny kodeliste ved å klikke på **Sett opp egne alternativer** og **Legg til alternativ**.
+5. Klikk på **Radioknapp 1** og sett verdien til ```true``` og ledeteksten til ```Ja```.
+6. Klikk på **Legg til alternativ** igjen.
+7. Klikk på **Radioknapp 2** og sett verdien til ```false``` og ledeteksten til ```Nei```.
+   !["Radioknapper for bekrefte relevant app"](radioknapper_bekreftelse.png)
+8. Klikk på **Radioknapper** under **Datamodellknytninger**-menyen 
+9. Velg **KanBrukeSkjema** som datamodellknytning. 
+
+
+### Vis og skjul sider ved hjelp av logiske uttrykk
+Sidene **Side1** og **Arbeidsforhold** skal vises når brukeren har bekreftet at de oppfyller kriteriene for appen. Dette oppnås 
```

Comment:
```suggestion
Sidene **Side1** og **Arbeidsforhold** skal vises når brukeren har bekreftet at de oppfyller kriteriene for appen. Dette oppnås 
```Sidene **Side1** og **Arbeidsforhold** skal vises når brukerne har bekreftet at de oppfyller kriteriene for appen. Dette gjør du